src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -253,27 +250,21 @@ def __call__(
 
         pixel_values = self.image_processor(
             images,
-            do_resize=do_resize,
-            do_normalize=do_normalize,
-            return_tensors=return_tensors,
-            image_mean=image_mean,
-            image_std=image_std,
-            input_data_format=input_data_format,
-            data_format=data_format,
-            resample=resample,
-            do_convert_rgb=do_convert_rgb,
+            **kwargs["image_kwargs"],
         )["pixel_values"]
 
+        max_length = kwargs.get("max_length", None)
```

Comment:
not sure this will work, since it's not guaranteed `max_length` will be in kwargs and not `text_kwargs`. IMO we should first merge the kwargs and then try to get attributes like `suffix` or `max_length`